spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -271,6 +273,19 @@ def reset_namespace(self, warning=False, silent=True, message=False):
                                             "\n<hr>"),
                                           before_prompt=False)
                     self.silent_execute("%reset -f")
+                    if kernel_env.get('SPY_AUTOLOAD_PYLAB_O') == 'True':
+                        self.silent_execute("from pylab import *")
+                    if kernel_env.get('SPY_SYMPY_O') == 'True':
+                        sympy_init = """
+                            from __future__ import division
+                            from sympy import *
+                            x, y, z, t = symbols('x y z t')
+                            k, m, n = symbols('k m n', integer=True)
+                            f, g, h = symbols('f g h', cls=Function)
+                            init_printing()"""
+                        self.silent_execute(dedent(sympy_init))
+                    if kernel_env.get('SPY_RUN_CYTHON') == 'True':
+                        self.silent_execute("%reload_ext Cython")
                     self.refresh_namespacebrowser()
                 else:
```

Comment:
> I didn't implement anything here because as far as I can see silent is always true

Ok, since that seems the case, please remove the use of `silent` in this method.